powa/static/js/components/Graph.vue

```diff
@@ -680,8 +681,10 @@ function pointermoved(event) {
     .selectAll("circle")
     .data(markersData)
     .attr("fill", (d) => getColor(d[0]))
-    .attr("display", null)
-    .attr("transform", (d) => `translate(${xScale(X[i])}, ${d[1]})`);
+    .attr("display", (d) => (d[1] ? null : "none"))
```

Comment:
Why not use isNil here? I don't understand what d[1] may contain.

Reply:
Yep. OK, let's be consistent.